tests/matrix_free/laplace_operator_02.cc

```diff
@@ -0,0 +1,291 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+
+// this tests the correctness of MPI-parallel matrix free matrix-vector
+// products using MatrixFreeOperators::LaplaceOperator by comparing the result
+// with a Trilinos sparse matrix assembled in the usual way. The mesh is
+// distributed among processors (hypercube) and has both hanging nodes
+// (by randomly refining some cells, so the mesh is going to be different
+// when run with different numbers of processors) and Dirichlet boundary
+// conditions
+//
+// this test does not use multithreading within the MPI nodes.
```

Comment:
Please remove this old text but say that this test is the same as `laplace_operator_01` but using a variable coefficient.